Fortnox.NET/Models/Account/Account.cs

```diff
@@ -9,7 +9,6 @@ public class Account : AccountSubset
         public bool Active { get; set; }
         public decimal BalanceBroughtForward { get; set; }
         public string Description { get; set; }
-        public int Number { get; set; }
```

Comment:
Should this really be removed? I looked in the single resource and it seems like it contains Number as well.

Reply:
It's been moved from here to the single resource.

Reply:
Ah sorry I didn't see `public class Account : AccountSubset`. Great job!